test/test_search.rb

```diff
@@ -0,0 +1,96 @@
+# ~*~ encoding: utf-8 ~*~
+path = File.join(File.dirname(__FILE__), 'helper')
+require File.expand_path(path)
+
+context 'Wiki search' do
+
+  SEARCH_TEST_PAGE = <<EOF
+Hobbits are small. Hobbits live in Hobbiton.
+One thing Hobbits do not like is Orcs.
+Something a hobbit does like is pastry.
+Some other words.
```

Comment:
How about some utf-8 instead? ` ᚠᛇᚻ᛫ᛒᛦᚦ᛫ᚠᚱᚩᚠᚢᚱ᛫ᚠᛁᚱᚪ᛫ᚷᛖᚻᚹᛦᛚᚳᚢᛗ`

Reply:
So geeky! I like it.